addons/mail/static/src/components/discuss/discuss.xml

```diff
@@ -8,26 +8,28 @@
                 'o-mobile': env.messaging.device.isMobile,
             }"
         >
-            <t t-if="!env.messaging.isInitialized">
-                <div class="o_Discuss_messagingNotInitialized"><i class="o_Discuss_messagingNotInitializedIcon fa fa-spinner fa-spin"/>Please wait...</div>
-            </t>
-            <t t-else="">
-                <t t-if="!env.messaging.device.isMobile">
-                    <DiscussSidebar class="o_Discuss_sidebar"/>
+            <t t-if="env.messaging">
+                <t t-if="!env.messaging.isInitialized">
+                    <div class="o_Discuss_messagingNotInitialized"><i class="o_Discuss_messagingNotInitializedIcon fa fa-spinner fa-spin"/>Please wait...</div>
```

Comment:
Don't we want also the spinner when `!env.messaging`, having a single `t-if` for both conditions?